### PR TITLE
Adding missing lookup fields for LDAP configuration in UI

### PR DESCRIPTION
**Pull request by Alevsk**
# Console
image

# Operator
image

Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Configurations/utils.tsx [hunk 1]
```diff
@@ -417,29 +417,45 @@ export const fieldsConfigurations: any = {
...
-      required: true,
+      required: false,
       label: "Username Search Filter",
       tooltip:
         'User search filter, for example "(cn=%s)" or "(sAMAccountName=%s)" or "(uid=%s)"',
       type: "string",
       placeholder: "Enter Username Search Filter",
     },
+    {
+      name: "user_dn_search_base_dn=",
```
Comment (harshavardhana) [inline portal-ui/src/screens/Console/Configurations/utils.tsx:428]:
```suggestion
      name: "user_dn_search_base_dn",
```

having `=` looks wrong

File: portal-ui/src/screens/Console/Configurations/utils.tsx [hunk 2]
```diff
@@ -417,29 +417,45 @@ export const fieldsConfigurations: any = {
...
+      required: false,
+      label: "User DN Search Base DN",
+      tooltip: "List of username search DNs, you can write one per field",
+      type: "csv",
+      placeholder: "Enter User DN Search Base DN",
+    },
+    {
+      name: "user_dn_search_filter=",
```
Comment (harshavardhana) [inline portal-ui/src/screens/Console/Configurations/utils.tsx:436]:
```suggestion
      name: "user_dn_search_filter",
```

having `=` looks wrong

File: portal-ui/src/screens/Console/Configurations/utils.tsx [hunk 3]
```diff
@@ -449,6 +465,22 @@ export const fieldsConfigurations: any = {
...
+      placeholder: "Enter Lookup Bind DN",
+    },
+    {
+      name: "lookup_bind_password",
+      required: false,
+      label: "Lookup Bind Password",
+      tooltip: "",
+      type: "string",
+      placeholder: "Enter Lookup Lookup Bind Password",
+    },
```
Comment (harshavardhana) [inline portal-ui/src/screens/Console/Configurations/utils.tsx:483]: these are required @Alevsk

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/Configurations/utils.tsx (hunk 3 above)
Comment (donatello) [reply portal-ui/src/screens/Console/Configurations/utils.tsx]: Lookup bind password is optional - we support unauthenticated bind too.

File: portal-ui/src/screens/Console/Configurations/utils.tsx [hunk 4]
```diff
@@ -404,75 +404,80 @@ export const fieldsConfigurations: any = {
...
+      name: "user_dn_search_filter",
       required: false,
-      label: "STS Expiry",
-      tooltip:
-        'temporary credentials validity duration in s,m,h,d. Default is "1h"',
+      label: "User DN Search Filter",
+      tooltip: "Search filter to lookup user DN",
       type: "string",
-      placeholder: "Enter STS Expiry",
+      placeholder: "(uid=%s,cn=accounts,dc=min,dc=io)",
```
Comment (donatello) [inline portal-ui/src/screens/Console/Configurations/utils.tsx:464]: This is not correct - this needs to be a search filter like: `"(sAMAcountName=%s)"

**Review by donatello — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Configurations/utils.tsx (hunk 4 above)
Comment (Alevsk) [reply portal-ui/src/screens/Console/Configurations/utils.tsx:464]: Hi @donatello I got this example from https://github.com/minio/minio/blob/master/docs/sts/ldap.md 

```
export MINIO_IDENTITY_LDAP_USER_DN_SEARCH_FILTER='(uid=%s,cn=accounts,dc=min,dc=io)'
```

I can change it to `"(sAMAcountName=%s)"` in console

**Review by harshavardhana — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]

**Pull request merged**